packages/core/src/types.ts

```diff
@@ -72,14 +72,14 @@ export type UserOperationPaymasterOverrides<
   TEntryPointVersion extends EntryPointVersion = EntryPointVersion
 > = TEntryPointVersion extends "0.6.0"
   ? {
-      // paymasterAndData overrides only allows '0x' for bypassing paymaster middleware
-      paymasterAndData: EmptyHex;
+      // paymasterAndData overrides allow either '0x' or the data to overwrite for bypassing paymaster middleware
+      paymasterAndData: Hex;
     }
   : TEntryPointVersion extends "0.7.0"
   ? {
-      // paymasterData overrides only allows '0x' for bypassing the paymaster middleware
+      // paymasterData overrides allow either '0x' or the data to overwrite for bypassing paymaster middleware
       // if set to '0x', all paymaster related fields are omitted from the user op request
-      paymasterData: EmptyHex;
+      paymasterData: Hex;
```

Comment:
oh we need to add `paymaster: Address` as a field here now too for when they're not setting the above to `0x`

Reply:
Should be good in https://github.com/alchemyplatform/aa-sdk/pull/676/commits/0de5caed2cac32f7395fdf46ab9d4282962a46a0!